asset-transfer-basic/chaincode-javascript/lib/assetTransfer.js

```diff
@@ -64,7 +64,7 @@ class AssetTransfer extends Contract {
     }
 
     // createAsset issues a new asset to the world state with given details.
-    async createAsset(ctx, id, color, size, owner, appraisedValue) {
+    async createAsset(ctx, id, color, owner, size, appraisedValue) {
```

Comment:
`createAsset` param sequence updated to match the chaincode-go `CreateAsset` args

Reply:
If you switch it back, it should be correct if they merge my pr #227.

Reply:
To match across all chaincodes and applications regardless of language, the order should be: id, color, size, owner, appraisedValue.

Reply:
I have also reconciled applications (typescript/javascript) to chaincodes.  See also PR's #228 and #229.